### PR TITLE
Replica files

### PR DESCRIPTION
**Pull request by iamchristianani**
## Upload the files for the YouTube replica
I was able to do the following
- Added the index.html file with the following things achieved
  - Embedded a YouTube video in my page
  - Made use of images in my page.
  - Made use of bootstrap for page styling
- Added the style.css folder with the following things achieved
  - Made use of floats
  - Made the page responsive by using a min-width
- Added the img folder with a few images
- Added a Javascript folder with Javascript files to achieve the following
  - Scroll feature on the sidebar (script.js)
  - To add icons to the page (feather.min.js)
  - Bootstrap usage

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+    margin-left: 0;
+}
+
+label {
+    margin-left: 10px;
+}
+
+.hide {
+    display: none;
+}*/
```
Comment (iamchristianani) [inline css/styles.css:178]: Please do not add unused codes as comments to your codebase so as to make your code look clean.

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+.hide {
+    display: none;
+}
+ 
+
+
+
+
+
+
```
Comment (iamchristianani) [inline css/styles.css:234]: Please make sure to remove the irrelevant spacing in your codebase. A space of just 1 line would be enough and help make your code look clean.

File: css/styles.css [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+}
+
+#right-button {
+    float: left;
+    width: 30px;
+    text-align: center;
+}
+
+  
+
```
Comment (iamchristianani) [inline css/styles.css:204]: Please make sure to remove the irrelevant spacing in your codebase. A space of just 1 line would be enough and help make your code look clean.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,490 @@
...
+                    <div class="col-md-1">
+                        <ul class="navbar-nav flex-row flex-wrap ms-md-auto">
+                            <li class="nav-item col-4 mt-1">
+                                <i data-feather="video" color="white"></i>
+                            </li>
+                            <li class="nav-item col-4 mt-1">
+                                <i data-feather="bell" color="white"></i>
+                            </li>
+                            <li class="nav-item col-4 mt-1">
+                                <img src="img/trey-songz.jpg" alt="Trey Songz" class="rounded-circle" style="width: 35px; height: 35px;">
```
Comment (iamchristianani) [inline index.html:45]: Please do not use inline styles. Keep your style definitions in the external CSS file you created.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,490 @@
...
+                                    <i data-feather="plus-square" class="mb-1"></i> Save
+                                </li>
+                            </ul>
+                        </div>
+                        <div class="creator-details mt-3">
+                            <div class="container">
+                                <div class="row justify-content-between">
+                                    <div class="d-flex justify-content-between col-md-12 col-sm-12 col-lg-12 col-xl-5 col-xxl-5 border border-secondary one-of-two-box p-1">
+                                        <div class="creator-left col-6 d-flex justify-content-start">
+                                            <img src="img/trey-songz.jpg" alt="Trey Songz" class="rounded-circle m-auto" style="width: 40px; height: 40px;">
```
Comment (iamchristianani) [inline index.html:96]: Please do not use inline styles. Keep your style definitions in the external CSS file you created.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,490 @@
...
+                                            </div>
+                                        </div>
+                                    </div>
+                                    <div class="d-flex justify-content-between col-md-12 col-sm-12 col-lg-12 border border-secondary one-of-two-box p-1">
+                                        <div class="col-4 comment-and-likes m-auto">
+                                            <span class="">Comments</span> <br>
+                                            <span style="font-size: 10px;" class="">1.1K</span>
+                                        </div>
+                                        <div class="col-8 d-flex justify-content-start">
+                                            <img src="img/trey-songz.jpg" alt="Trey Songz" class="rounded-circle mt-2" style="width: 25px; height: 25px;">
```
Comment (iamchristianani) [inline index.html:115]: Please do not use inline styles. Keep your style definitions in the external CSS file you created.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,490 @@
...
+                            <div class="comment-no-and-sorts d-flex justify-content-start">
+                                <div class="me-5">1,123 Comments</div>
+                                <div class="d-flex justify-content-start">
+                                    <i data-feather="align-left"></i>
+                                    <p class="ms-2">SORT BY</p>
+                                </div>
+                            </div>
+                            <div class="form-section d-flex justify-content-start">
+                                <div class="commenter-image col-1">
+                                    <img src="img/trey-songz.jpg" alt="Trey Songz" class="rounded-circle m-auto" style="width: 40px; height: 40px;">
```
Comment (iamchristianani) [inline index.html:136]: Please do not use inline styles. Keep your style definitions in the external CSS file you created.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,490 @@
...
+                                            <button type="button" class="btn btn-dark cancel-button">CANCEL</button>
+                                            <button type="button" class="btn btn-primary ms-3">COMMENT</button>
+                                        </div>
+                                    </form>
+                                </div>
+                            </div>
+                            <div class="users-comments">
+                                <div class="ind-user-comment d-flex justify-content-start">
+                                    <div class="commenter-image col-1">
+                                        <img src="img/trey-songz.jpg" alt="Trey Songz" class="rounded-circle m-auto" style="width: 40px; height: 40px;">
```
Comment (iamchristianani) [inline index.html:152]: Please do not use inline styles. Keep your style definitions in the external CSS file you created.

File: index.html [hunk 9]
```diff
@@ -0,0 +1,490 @@
...
+                                    </div>
+                                    <div class="form-box col-11">
+                                        <div class="user-info">
+                                            <p style="font-size: 15px;">John Richard <span style="color: #a0a0a0; font-size: 12px;">1 year ago</span></p>
```
Comment (iamchristianani) [inline index.html:156]: Please do not use inline styles. Keep your style definitions in the external CSS file you created.

File: index.html [hunk 10]
```diff
@@ -0,0 +1,490 @@
...
+                                        </div>
+                                    </div>
+                                </div>
+                                <div class="ind-user-comment d-flex justify-content-start mt-4">
+                                    <div class="commenter-image col-1">
+                                        <img src="img/trey-songz.jpg" alt="Trey Songz" class="rounded-circle m-auto" style="width: 40px; height: 40px;">
+                                    </div>
+                                    <div class="form-box col-11">
+                                        <div class="user-info">
+                                            <p style="font-size: 15px;">John Richard <span style="color: #a0a0a0; font-size: 12px;">1 year ago</span></p>
```
Comment (iamchristianani) [inline index.html:177]: Please do not use inline styles. Keep your style definitions in the external CSS file you created.

File: index.html [hunk 11]
```diff
@@ -0,0 +1,490 @@
...
```
Comment (iamchristianani) [inline index.html:198]: Please do not use inline styles. Keep your style definitions in the external CSS file you created.

**Review by iamchristianani — COMMENTED** [review]
Hi @iamchristianani , 

Good job so far!
These are a few changes that you still need to work on to make this project a great one, but you are almost there!

## Highlights
📌 Descriptive pull request✅
📌 Project working as expected ✅
📌 Good commit messages ✅ 
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._
- [ ] Please kindly make use of flexboxes and grids to arrange the webpage properly. 😃 
- [ ] For readability purposes, I will suggest that you make use of just 2 spaces for indentations, and not tabs (On some code editors, you can set the tab to an x amount of spaces). 😃 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Review by nedjwab — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @iamchristianani  👋

Your project is complete! ✔️ 🏁

Your code was very professional 💯

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉

 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**